spec/views/cases/new_html_slim_spec.rb

```diff
@@ -98,7 +105,7 @@
       expect(page).to have_external_deadline_month
       expect(page).to have_external_deadline_year
 
-      expect(page).to have_subject
+      expect(page).not_to have_subject
```

Comment:
could this be `expect(page).to have_no_subject` i believe using `have_no` is more performant than using `not_to`

Reply:
Sure, will change it.